src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -86,6 +96,26 @@ private void bindHttpServer(Future<Void> startFuture) {
                 });
     }
 
+    private void startInactiveConsumerDeletionTimer(Long timeout) {
+        Long timeoutInMs = timeout * 1000L;
+        vertx.setPeriodic(1000, ignore -> {
```

Comment:
I ask again, does this really need to run every second?  Why 1 second?

Reply:
I used the polling value we are using for another operation. 
Of course, we can use another value but what it should be? Should it be related to the timeout?

Reply:
I would use something like ½ the timeout. Remember, this is a defence against badly written clients, so it doesn't matter if a client consumer is closed "late" (i.e. after the timeout), so long as it is closed. We also expect that this actually ends up closing a consumer only rarely, so it's not worth spending cycles on trying to be accurate.